### PR TITLE
optimize and more like Scala

### PR DESCRIPTION
**Pull request by jxnu-liguobin**
## What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

### CODE REVIEW

**Review by wey-gu — APPROVED** [review]

File: nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/lib/AlgorithmType.scala [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package com.vesoft.nebula.algorithm.lib
```
Comment (Nicole00) [inline nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/lib/AlgorithmType.scala:1]: please add the copyright

File: nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/reader/ReaderType.scala [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package com.vesoft.nebula.algorithm.reader
```
Comment (Nicole00) [inline nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/reader/ReaderType.scala:1]: ditto

File: nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/writer/WriterType.scala [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package com.vesoft.nebula.algorithm.writer
```
Comment (Nicole00) [inline nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/writer/WriterType.scala:1]: ditto

**Review by Nicole00 — APPROVED** [review]
LGTM! Thanks for your contribution~

**Pull request merged**